app/ui/src/main/java/com/fsck/k9/ui/managefolders/FolderSettingsFragment.kt

```diff
@@ -53,10 +80,44 @@ class FolderSettingsFragment : PreferenceFragmentCompat() {
         findPreference<Preference>(PREFERENCE_TOP_CATEGORY)!!.title = folderDisplayName
     }
 
+    private fun handleActionEvents(action: Action) {
+        when (action) {
+            is Action.ClearFolder -> onClearFolder()
+        }
+    }
+
+    private fun onClearFolder() {
+        val dialogFragment = ConfirmationDialogFragment.newInstance(
+                DIALOG_CLEAR_FOLDER,
+                getString(R.string.dialog_confirm_delete_title),
```

Comment:
These seem to be the wrong strings. The title is "Confirm deletion", the messasge "Do you want to delete this message?".